cmd/kogito/command/message/data_index.go

```diff
@@ -22,7 +22,7 @@ var (
 	// DataIndexSuccessfulInstalled ...
 	DataIndexSuccessfulInstalled = fmt.Sprintf(serviceSuccessfulInstalled, "Data Index", "%s")
 	// DataIndexCheckStatus ...
-	DataIndexCheckStatus = fmt.Sprintf(serviceCheckStatus, "kogitodataindex", "%s", "%s")
+	DataIndexCheckStatus = fmt.Sprintf(serviceCheckStatus, "kogitosupportingservice", "%s", "%s")
```

Comment:
shouldn't this be moved some place else ? it is no more dataindex related

Reply:
Yes, missed this, it'll be moved for all the services, in a single file